Library/Homebrew/utils/popen.rb

```diff
@@ -1,20 +1,22 @@
 module Utils
   def self.popen_read(*args, &block)
-    popen(args, "rb", &block)
+    options = args.last.is_a?(Hash) ? args.pop : {}
```

Comment:
Instead of this, you can use the double splat operator: 

```ruby
def self.popen_read(*args, **options, &block)
```

Reply:
Nice! Thanks, Markus. I've updated the PR.